aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -231,18 +286,60 @@ Tensor linalg_pinv(const Tensor& input, const Tensor& rcond, bool hermitian) {
     Tensor S_abs = S.abs();
```

Comment:
It should be squared in this case, as the singular values are the squared norm of the eigenvalues.

Reply:
The code is correct, the singular values are the absolute values of the eigenvalues of a Hermitian matrix.

Reply:
Ah true, you are completely right.